kolibri/plugins/learn/assets/src/views/LearnExamReportViewer.vue

```diff
@@ -24,8 +24,13 @@
           :navigateTo="navigateTo"
           :questions="questions"
           :sections="exam.question_sources"
+          :isQuizActive="exam.active"
           @noCompleteTries="noCompleteTries"
-        />
+        >
+          <template #unStartedQuiz>
```

Comment:
I do not think we need this string here but I have added in case there is any scenario of learner accessing un started quiz that I might have not seen.